### DISCUSSION TITLE
荣耀新系统gkd无法获取无障碍权限

### DISCUSSION THREAD

**Discussion by TG6** [gkd-kit/gkd#432] [Q&A]
我手机是荣耀magic5pro，系统是magic os 8.0，之前gkd更新1.6后，无障碍权限掉了，尝试恢复也不行，一点开启就会自动关 闭无障碍权限（安装旧版本是这样的情况）[log.zip](https://github.com/gkd-kit/gkd/files/13787336/log.zip)

**Comment by lisonge** [comment]
1.6.2也是这样吗？

**Reply by TG6** [reply]
> 1.6.2也是这样吗？

是的

**Reply by lisonge** [reply]
这个日志是不是有点问题，你卸载重新安装 1.6.2 看看呢

**Reply by TG6** [reply]
> 这个日志是不是有点问题，你卸载重新安装 1.6.2 看看呢

卸载重新安装之后，还是这样[log.zip](https://github.com/gkd-kit/gkd/files/13787673/log.zip)

**Reply by lisonge** [reply]
不好意思，你想表达什么？我在日志里都没有看到你去申请无障碍权限，只看到你打开了两次界面

**Reply by B0ILEDWATER** [reply]
oppo的c14系统，GKD更新1.6.2之后也是无障碍提示因为安全问题不让GKD获取权限了，再降级回1.6.0之后也不行了

**Reply by lisonge** [reply]
查看置顶 issue
